src/models/game.spec.js

```diff
@@ -442,4 +442,132 @@ describe("game.js", function () {
       });
     });
   });
+
+  describe(".updateMissingProperties updates the game with the provided properties.", function () {
+    describe("When all the properties are provided", function () {
+      beforeEach(function () {
+        const steamApp = {
```

Comment:
extract mock data into methods, you can remove a lot of duplication in this tests